public/steelthreads/json/en/thread-2.json

```diff
@@ -0,0 +1,53 @@
+{
+  "id": 1,
+  "title": "Integrating event-driven and API-driven applications",
+  "descriptionDoc": "integrating-eventdriven-and-apidriven-applications-introduction.adoc",
+  "attributes": {
+    "fuse-url": "https://cluster-lfa3xlh.opentry.me"
+  },
+  
+  "prerequisites": [
+    "You have completed Walkthrough 0"
+  ],
+  "estimatedTime": 50,
+  "roles": [
+    "Operator",
+    "Developer"
+  ],
+  "applications": [
+    "Fuse Ignite",
+    "EnMasse"
+  ],
+  "tasks": [
+    {
+      "title": "Creating an integration",
+      "estimatedTime": 5,
+      "steps": [
+        {
+          "stepDoc": "creating-fuse-integration.adoc",
+          "infoVerifications": [
+            "creating-fuse-integration-verification.adoc"
+          ],
+          "attributes": {
+            "fuse-url": "https://cluster-lfa3xlh.opentry.me"
```

Comment:
Which of these `fuse-url`s should take precedence? The top guy or this guy?

Reply:
I'll remove 'this guy'